scripts/build_benchmark_configs.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/sh
+
+# Helper script to build multiple configurations of a benchmark.  Patches the
+# board description files to enable and disable different combinations of
+# hardware features.  Also patches include paths to keep track of the original
+# locations.
+
+set -eo pipefail
```

Comment:
Debian's `dash`, which it uses as `/bin/sh`, apparently doesn't like `-o pipefail`, which seems surprising.

Reply:
Hmm, not sure if it’s better to require bash or skip the pipe fail here. It should be fine with zsh and, I think, the FreeBSD POSIX shell.

Reply:
Are we actually doing anything in a pipeline in this script?  The only `|`s I see are either part of `||`s or inside `jq` expressions.

Reply:
Hmm, maybe not, so just -eo should be fine.

Reply:
Just `-e`, I think?  And`-u` is also generally a good plan, but I haven't looked for where we might rely on not-that at the moment.